src/Tasks/Microsoft.NET.Build.Tasks/msbuildExtensions/Microsoft.NET.Build.Extensions/Microsoft.NET.Build.Extensions.ConflictResolution.targets

```diff
@@ -0,0 +1,55 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Choose>
+    <!-- Allow completely disabling the conflict resolution targets -->
+    <When Condition="'$(DisableHandlePackageFileConflicts)' == 'true'" />
+    <!-- SDK projects have built in conflict resolution that understands assets/deps file -->
+    <When Condition="'$(UsingMicrosoftNETSdk)' == 'true'" />
+    <When Condition="'$(TargetFramework)' != '' or '$(TargetFrameworks)' != ''">
+      <!-- SDK project missing inbox conflict resolution.  Emit an error -->
+      <PropertyGroup>
+        <PackagingToolsInsufficientSDKVersionAfter>ResolvePackageDependenciesForBuild</PackagingToolsInsufficientSDKVersionAfter>
+      </PropertyGroup>
+    </When>
+    <When Condition="'$(ResolveNuGetPackages)' == 'true' and Exists('$(ProjectLockFile)')">
+      <!-- Non-SDK using project.json or PackageReference, run after references are read from the lock/assets file -->
+      <PropertyGroup>
+        <HandlePackageFileConflictsAfter>ResolveNuGetPackageAssets</HandlePackageFileConflictsAfter>
+      </PropertyGroup>
+    </When>
+    <Otherwise>
+      <!-- No specific NuGet targets to sequence after, run before targets that consume references -->
+      <PropertyGroup>
+        <ResolveAssemblyReferencesDependsOn>$(ResolveAssemblyReferencesDependsOn);HandlePackageFileConflicts</ResolveAssemblyReferencesDependsOn>
+        <PrepareResourcesDependsOn>HandlePackageFileConflicts;$(PrepareResourcesDependsOn)</PrepareResourcesDependsOn>
+      </PropertyGroup>
+    </Otherwise>
+  </Choose>
+
+  <Target Name="PackagingToolsInsufficientSDKVersion" Condition="'$(PackagingToolsInsufficientSDKVersion)' != 'false'" AfterTargets="$(PackagingToolsInsufficientSDKVersionAfter)">
+    <Error Text="The version of Microsoft.NET.Sdk used by this project is insufficient to support this package.  Please ensure you are using the .NET Core SDK 2.0 or higher, which is included in Visual Studio 2017 Update 15.3.  This error may be suppressed by setting PackagingToolsInsufficientSDKVersion=false." />
```

Comment:
VS 15.3 won't necessarily ship with the .NET Core SDK in-box, it may only ship with a 1.1 version of the SDK.  So if you haven't installed the 2.0 SDK, I think as is you'll always get this error message when you try to build an sdk-style project using full MSBuild.

On the other hand, if you haven't installed the 2.0 SDK but are targeting .NET 4.6.1, we should either make it work, or provide a good error.  We need to figure out when & how to raise the error, and the error text needs to be changed (since the 2.0 SDK won't necessarily be in VS 15.3) and put into a .resx for localization (which we do with the NETSdkError task).